src/main/java/Gops/Gops.java

```diff
@@ -0,0 +1,142 @@
+package Gops;
+
+import javafx.scene.image.Image;
+import javafx.scene.image.ImageView;
+import javafx.application.Application;
+import javafx.stage.Stage;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.control.ScrollPane;
+import javafx.scene.control.TextField;
+import javafx.scene.layout.AnchorPane;
+import javafx.scene.layout.VBox;
+import javafx.scene.layout.Region;
+import javafx.scene.control.Label;
+import java.io.File;
+import java.io.IOException;
+import java.util.Scanner;
+
+public class Gops extends Application {
```

Comment:
Should public classes have JavaDoc header comments? I noticed the same issue in other files too.